bin/node-template/pallets/template/src/mock.rs

```diff
@@ -23,7 +23,6 @@ frame_support::construct_runtime!(
 );
 
 parameter_types! {
-	pub const BlockHashCount: u64 = 250;
 	pub const SS58Prefix: u8 = 42;
```

Comment:
this one should also be removable with `ConstU8<42>`.

Reply:
Alright.